serviceapis/twitterapi.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+'''
+Copyright © 2017, ACM@UIUC
+FoodButton is open source software, released under the University of
+Illinois/NCSA Open Source License.  You should have received a copy of
+this license in a file with the distribution.
+'''
+
+from .serviceapi import ServiceAPI
+from datetime import datetime
+import tweepy
+from PIL import Image
+
+class TwitterAPI(ServiceAPI):
+    '''
+    Implements the ServiceAPI interface for Twitter
+    '''
+    def __init__(self, cfg):
+        auth = tweepy.OAuthHandler(cfg['consumer_key'], cfg['consumer_secret'])
+        auth.set_access_token(cfg['access_token'], cfg['access_token_secret'])
+        self.api = tweepy.API(auth)
+         
+    def post(self, img, text):
+        try:
+            filename = "/tmp/food_on_table_{}.png".format(datetime.now())
+            img.save(filename)
+            self.api.update_with_media(filename, status=text)
+        except tweepy.TweepError as tweepError:
+            print (tweepError.reason)  #TODO: Change to some sort of logger 
```

Comment:
I'd using the standard python `logging` module here. Though, this can be done in a later PR

Reply:
Ok will file an issue for this

Reply:
I can do this now.